### ISSUE TITLE
2021-9-25

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#613]
451Now we just have to run that with a bunch of examples. Since the train function returns both the output and loss we can print its guesses and also keep track of loss for plotting. Since there are 1000s of examples we print only every print_every examples, and take an average of the loss. (2021-9-20)
452Change line_tensor to name_tensor; done; replace guess_i with guess_index; done; (2021-9-20)
453Add function add_random_pair; done; add function get_pair_tensor; done; (2021-9-21)
454Add [(),()]; done; (2021-9-21)
455How to shuffle list in python? Random.shuffle(mylist); (2021-9-21)
456一个epoch 指的是把所有训练数据丢进神经网络一次; add epoch to program; done; (2021-9-21)
457Reduce the learning rate to 0.003 to see the result; not improved; (2021-9-21)
458Use optimizer to replace rnn.parameters() ; done; (2021-9-21)
459交叉熵代表损失函数; 损失函数越小越好, 交叉熵自然也是越小越好; (2021-9-22)
